pep-0480.txt

```diff
@@ -23,13 +23,23 @@ developers to sign for the distributions that are downloaded by clients.  The
 minimum security model proposed by PEP 458 supports continuous delivery of
 distributions (because they are signed by online keys), but that model does not
 protect distributions in the event that PyPI is compromised.  In the minimum
-security model, attackers may sign for malicious distributions by compromising
-the signing keys stored on PyPI infrastructure.   The maximum security model,
+security model, attackers who have compromised the signing keys stored on PyPI
+Infrastructure may sign for malicious distributions.   The maximum security model,
 described in this PEP, retains the benefits of PEP 458 (e.g., immediate
 availability of distributions that are uploaded to PyPI), but additionally
 ensures that end-users are not at risk of installing forged software if PyPI is
 compromised.
 
+This PEP requires some changes to the PyPI infrastructure, and some suggested
+changes for developers who wish to participate in end-to-end signing. These
+changes include updating the metadata layout from PEP 458 to include delegations
+to developer keys, adding a process to register developer keys with PyPI, and a
+change in the upload workflow for developers who take advantage of end-to-end
+signing. All of these changes are described in detail later in this PEP. Package
+managers that wish to take advantage of end-to-end signing do not need to do any
```

Comment:
With this all in one paragraph, it's easy to miss that the last sentence is about clients rather than the backend infrastructure. I'd suggest make it a separate paragraph and emphasising the `*not*`